src/elements/mesh/mesh_configuration.rs

```diff
@@ -0,0 +1,240 @@
+                $mesh_configuration_property_name:ident => $mesh_configuration_property_type:expr
+            ),*
+        }
+    ) => {
+        $(
+            #[$enum_meta]
+        )*
+        #[non_exhaustive]
+        $enum_vis enum $enum_name {
+            Unknown {
+                mesh_configuration_property_selector: u8
+            },
+            $(
+                $(
+                    #[$mesh_configuration_property_meta]
+                )*
+                $mesh_configuration_property_name
+            ),*
+        }
+        impl $enum_name {
+
+            #[inline]
+            pub const fn with_mesh_configuration_property_selector(mesh_configuration_property_selector: u8) -> Self {
+                match mesh_configuration_property_selector {
+                    $(
+                        $mesh_configuration_property_type => Self::$mesh_configuration_property_name,
+                    )*
+                    identifier => Self::Unknown {
+                        mesh_configuration_property_selector: identifier
+                    }
+                }
+            }
+            #[inline]
+            pub const fn mesh_configuration_property_selector(&self) -> u8 {
+                match *self {
+                    $(
+                        Self::$mesh_configuration_property_name => $mesh_configuration_property_type,
+                    )*
+                    Self::Unknown { mesh_configuration_property_selector } => mesh_configuration_property_selector
+                }
+            }
+        }
+        impl<'a> TryFromCtx<'a> for $enum_name {
+            type Error = scroll::Error;
+            #[inline]
+            fn try_from_ctx(from: &'a [u8], _ctx: ()) -> Result<(Self, usize), Self::Error> {
+                let mut offset = 0;
+
+                let mesh_configuration_property_selector = from.gread_with(&mut offset, Endian::Little)?;
+
+                Ok((
+                    Self::with_mesh_configuration_property_selector(mesh_configuration_property_selector),
+                    offset
+                ))
+            }
+        }
+        impl SizeWith for $enum_name {
+            #[inline]
+            fn size_with(_ctx: &()) -> usize {
+                1
+            }
+        }
+        impl TryIntoCtx for $enum_name {
+            type Error = scroll::Error;
+            #[inline]
+            fn try_into_ctx(self, buf: &mut [u8], _ctx: ()) -> Result<usize, Self::Error> {
+                let mut offset = 0;
+
+                buf.gwrite_with(self.mesh_configuration_property_selector(), &mut offset, Endian::Little)?;
+
+                Ok(offset)
+            }
+        }
+    };
+}
+
+mesh_configuration_property! {
+    #[cfg_attr(feature = "defmt", derive(defmt::Format))]
+    #[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Hash)]
+    pub enum MeshConfigurationActivePathSelectionProtocolIdentifier {
+        #[default]
+        HWMP => 1,
+        VendorSpecific => 255
+    }
+}
```

Comment:
I'd instead use `serializable_enum!` here, since it pretty much does the same, just without the `scroll` impls.